server/src/main/java/org/apache/druid/server/log/EmittingRequestLogger.java

```diff
@@ -56,96 +51,7 @@ public String toString()
     return "EmittingRequestLogger{" +
            "emitter=" + emitter +
            ", feed='" + feed + '\'' +
+           ", requestLogEventBuilderFactory=" + requestLogEventBuilderFactory +
```

Comment:
`DefaultRequestLogEventBuilderFactory.toString()` is not implemented.

Reply:
Thanks, fixed